net/src/icmp4/mod.rs

```diff
@@ -261,7 +261,7 @@ mod contract {
             let icmp_header = Icmpv4Header {
                 icmp_type: Icmpv4Type::DestinationUnreachable(
                     DestUnreachableHeader::from_values(
-                        driver.produce()?,
+                        driver.produce::<u8>()? % 16,
```

Comment:
Does the 16 have to be a magic number?  Do we have a constant that explains what it is?

Reply:
Not worth it, we'll keep the magic number for now.  I wish etherparse had a constant, but oh well.